pages/common/htop.md

```diff
@@ -1,19 +1,20 @@
 # htop
 
 > Display dynamic real-time information about running processes. An enhanced version of `top`.
+> More information: <https://htop.dev/>.
 
 - Start htop:
 
 `htop`
 
-- Start htop displaying only processes owned by given user:
+- Start htop displaying processes owned by a specific user:
 
-`htop -u {{username}}`
+`htop --username {{username}}`
 
-- Sort processes by a column (use `--sort-key help` for a column list):
+- Sort processes by a specified `sort_item` (use `htop --sort help` for available options):
 
-`htop -s {{column_name}}`
+`htop --sort {{sort_item}}`
 
-- Get help about interactive commands:
+- Display help:
 
-`?`
+`htop --help`
```

Comment:
It's ok if you want to add the `--help` example, but the `?` for the interactive mode is really helpful in this page.

Reply:
@CleanMachine1 how about adding both examples?

Reply:
hmm, I really wondered what it meant by ?

I thought the original creator had just put, "I don't know how to get help"

Maybe change the desc to something like 

```
- Whilst running htop, see interactive commands:
```

Reply:
No, in this case it's a key that you press when htop is actually running, which then displays the inbuilt help on how to use it. That description sounds ok though @CleanMachine1.

Reply:
I'll add it now.

Reply:
https://github.com/tldr-pages/tldr/pull/6165/commits/1af5616de26967028c09a19ccd4ef0ccf71bffb2